src/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/proxy.go

```diff
@@ -52,27 +53,27 @@ func (mod ProxyModifier) getVolumeMounts() []corev1.VolumeMount {
 	return []corev1.VolumeMount{
```

Comment:
nitpick: not related to your change really, but this is the same as 
```
return []corev1.VolumeMount{
		{
			Name:      config.ProxySecretVolumeName,
			MountPath: config.ProxySecretMountPath,
		},
	}
```
the same way you did it in the oneagent daemonset